### PR TITLE
[iOS] Fix issue using TapGesture

### LINKED ISSUE #10623: Tap Gesture not working on iOS  [Bug]
When having a tap gesture within a frame in xaml it will work on some but not all tap gestures. When adding it to the .cs file it would work but not directly within xaml. 

### Steps to Reproduce

1. 
2. 
3. 

### Basic Information

- Version with issue:
- Last known good version:
- IDE:
- Platform Target Frameworks: 
  - iOS:  
  - Android:  
  - UWP:   
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### LINKED ISSUE #12685: [iOs][Bug] TapGestureRecognizer in Path does not work on iOS
TapGestureRecognizer in Path does not work on iOS

`        <Grid BackgroundColor="Red">
            <Path Data="M0,37 V0 H193.86 L179.575,26.261 A19.5,19.5,0,0,1,162.148,37 Z">
                <Path.Fill>
                    <LinearGradientBrush>
                        <GradientStop Offset="0" Color="White" />
                        <GradientStop Offset="1" Color="Orange" />
                    </LinearGradientBrush>
                </Path.Fill>
                <Path.GestureRecognizers>
                    <TapGestureRecognizer Command="{Binding TestCommand}" />
                </Path.GestureRecognizers>
            </Path>
        </Grid>`

### Basic Information
Xamarin Forms 4.8 and 5.0 pre3:
Visual Studio 2019 Version 16.7.7

### LINKED ISSUE #13020: [Bug] TapGestureRecognizer Not working on Path on iOS
### Description

TapGestureRecognizer(setting on Path) does not work on iOS but it works on Android . 

### Steps to Reproduce

1. Create a simple Path and add TapGestureRecognizer 

```
  <Path Data="M 10,100 L 100,100 100,50Z"
      Stroke="Black"
      StrokeThickness="5"
      Aspect="Uniform"
      HorizontalOptions="Start" >
        <Path.GestureRecognizers>
            <TapGestureRecognizer Tapped="TapGestureRecognizer_Tapped"/>
        </Path.GestureRecognizers>
    </Path>

```
2. Click on the path .

### Expected Behavior

TapGestureRecognizer_Tapped event triggers

### Actual Behavior

TapGestureRecognizer_Tapped event not trigger

### Basic Information

Xamarin.Forms : 4.8.0.1687
Xamarin.iOS : 14.4.1.3

Test on iPhone 8 simulator

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Fix issue using TapGesture on iOS.

### Issues Resolved ### 

- fixes #10623
- fixes #12685
- fixes #13020

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

fix10623

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 10623. Tap the rating control. If you can modify the number of selected stars, the test has passed.

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by samhouts** [comment]
@jsuarezruiz The test needs to be updated to use brushes.

**Comment by jsuarezruiz** [comment]
Updated the test to use brushes.

**Comment by bholmes** [comment]
Wow you are kidding me? I do not commit to the repo for years, and one day I decide to pick off something that seems easy. I start to type the commit message and find that you have created this PR. 

Of course I am not really upset, just facepalming at the irony.  

The solution I had was not as good, I disabled user interaction on all the native views in the shape renderer class allowing events to pass through.

**Comment by bholmes** [comment]
Here is a copy of my test for this. Verified that the patch passes this test.

https://gist.github.com/bholmes/f7b68c31641e99019c6b11ee6286b575

**Comment by jsuarezruiz** [comment]
:( Sorry. Actually this PR was related to 10623 until recently I tried and added 12685 (which is exactly the same).
But, I have added your tests for 12685!

**Comment by rmarinho** [comment]
Failing test on iOS not related

**Review by hartez — APPROVED** [review]

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue10623.cs [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+#if UITEST
+	[Category(UITestCategories.Shape)]
+#endif
+    [Preserve(AllMembers = true)]
+    [Issue(IssueTracker.Github, 10623, "Tap Gesture not working on iOS [Bug]", PlatformAffected.iOS)]
+    public class Issue10623 : TestContentPage
+	{
+		public Issue10623()
+        {
+            Device.SetFlags(new List<string>(Device.Flags ?? new List<string>()) { "Shapes_Experimental" });
```
Comment (rmarinho) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue10623.cs:25]: think we don't need this right?

**Comment by rmarinho** [comment]
We should run all uitests as this test is not just on iOS

**Review by PureWeen — APPROVED** [review]

**Pull request merged**